packages/react/src/patterns/blocks/CalloutWithMedia/CalloutWithMedia.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { Callout } from '../../sub-patterns';
+import { ContentBlockSimple } from '../ContentBlockSimple';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings } from 'carbon-components';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ *
+ * @param {object} props Props object
+ * @param {object} props.contentblocksimple Contentblocksimple object
+ * @returns {*} Callout with media pattern
+ */
+const CalloutWithMedia = ({ contentblocksimple }) => {
+  return (
+    <div
+      data-autoid={`${stablePrefix}--callout-with-media`}
+      className={`${prefix}--callout-with-media`}>
+      <Callout>
+        <ContentBlockSimple inverse={true} {...contentblocksimple} />
+      </Callout>
+    </div>
+  );
+};
+
+CalloutWithMedia.propTypes = {
+  contentblocksimple: PropTypes.shape({
```

Comment:
@andysherman2121 I'm wondering if rather than having contentblocksimple as a prop, we should have the props be passed in directly?

```
const CalloutWithMedia = ({ ...ContentBlockSimpleProps }) => {
  ...
};

CalloutWithMedia.propTypes= PropTypes.oneOf(PropTypes.shape(ConentBlockSimple.propTypes));
```

Thoughts others? cc: @kennylam @annawen1 @reesedownes @guilhermelMoraes @greeshmareddyg

Reply:
I did it this way to match how we have Callout Quote